<ui version="4.0" >
 <class>DriverCardDialog</class>
 <widget class="QDialog" name="DriverCardDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/resources.qrc" >:/32/delivery.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>About:</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="4" >
           <widget class="QToolButton" name="addautotoolButton" >
            <property name="focusPolicy" >
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text" >
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QComboBox" name="autocomboBox" />
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Full name:*</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QDateEdit" name="borndateEdit" >
            <property name="calendarPopup" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="6" colspan="2" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Personal num:*</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5" colspan="2" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Nickname:*</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="5" >
           <widget class="QLineEdit" name="fullnamelineEdit" >
            <property name="minimumSize" >
             <size>
              <width>350</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="7" colspan="2" >
           <widget class="QLineEdit" name="nicklineEdit" />
          </item>
          <item row="0" column="8" >
           <widget class="QLineEdit" name="pnumlineEdit" >
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Auto:*</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Born:*</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Address:</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="0" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_13" >
              <property name="text" >
               <string>Phone:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="phonelineEdit" />
            </item>
            <item>
             <widget class="QLabel" name="label_14" >
              <property name="text" >
               <string>Mobile:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="moblineEdit" />
            </item>
           </layout>
          </item>
          <item row="0" column="0" >
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_10" >
              <property name="text" >
               <string>City:*</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="citycomboBox" >
              <property name="minimumSize" >
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimumContentsLength" >
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="addcitytoolButton" >
              <property name="focusPolicy" >
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text" >
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11" >
              <property name="text" >
               <string>Street:*</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="streetcomboBox" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="minimumSize" >
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="accessibleDescription" >
               <string/>
              </property>
              <property name="editable" >
               <bool>false</bool>
              </property>
              <property name="minimumContentsLength" >
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="addstreettoolButton" >
              <property name="focusPolicy" >
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text" >
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12" >
              <property name="text" >
               <string>Home â„–:*</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="homelineEdit" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Passport:</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="1" >
             <widget class="QLineEdit" name="passnumlineEdit" />
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_9" >
              <property name="text" >
               <string>Number:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>Serial:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QDateEdit" name="passdateEdit" >
              <property name="calendarPopup" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="passerlineEdit" />
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_16" >
              <property name="text" >
               <string>Date:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_15" >
            <property name="text" >
             <string>Get out:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="getoutlineEdit" />
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item rowspan="2" row="0" column="0" >
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="minimumSize" >
            <size>
             <width>128</width>
             <height>128</height>
            </size>
           </property>
           <property name="title" >
            <string>Photo:</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="photolabel" >
              <property name="maximumSize" >
               <size>
                <width>128</width>
                <height>128</height>
               </size>
              </property>
              <property name="text" >
               <string>no photo:</string>
              </property>
              <property name="scaledContents" >
               <bool>false</bool>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QToolButton" name="addphototoolButton" >
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>21</width>
           <height>62</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="savepushButton" >
         <property name="focusPolicy" >
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text" >
          <string>Save</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelpushButton" >
         <property name="focusPolicy" >
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text" >
          <string>Cancel</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fullnamelineEdit</tabstop>
  <tabstop>pnumlineEdit</tabstop>
  <tabstop>borndateEdit</tabstop>
  <tabstop>autocomboBox</tabstop>
  <tabstop>addautotoolButton</tabstop>
  <tabstop>nicklineEdit</tabstop>
  <tabstop>citycomboBox</tabstop>
  <tabstop>streetcomboBox</tabstop>
  <tabstop>addstreettoolButton</tabstop>
  <tabstop>homelineEdit</tabstop>
  <tabstop>phonelineEdit</tabstop>
  <tabstop>moblineEdit</tabstop>
  <tabstop>passerlineEdit</tabstop>
  <tabstop>passnumlineEdit</tabstop>
  <tabstop>passdateEdit</tabstop>
  <tabstop>getoutlineEdit</tabstop>
  <tabstop>savepushButton</tabstop>
  <tabstop>cancelpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cancelpushButton</sender>
   <signal>clicked()</signal>
   <receiver>DriverCardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>766</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel" >
     <x>516</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
